pkg/controller/clusters/status.go

```diff
@@ -81,7 +81,11 @@ func updateConditionsForEtcdAndControlPlane(cluster *anywherev1.Cluster, kcp *co
 	}
 	// Make sure etcd machine is ready before marking ControlPlaneReady status to true
 	if cluster.Spec.ExternalEtcdConfiguration != nil && !etcdadmClusterReady(etcdadmCluster) {
-		conditions.MarkFalse(cluster, anywherev1.ControlPlaneReadyCondition, anywherev1.RollingUpgradeInProgress, clusterv1.ConditionSeverityInfo, "Etcd is not ready")
+		if cluster.Spec.ControlPlaneConfiguration.UpgradeRolloutStrategy.Type == anywherev1.InPlaceStrategyType {
```

Comment:
how do you feel about this instead.?
```suggestion 
                upgradeStrategyCondition := anywherev1.RollingUpgradeInProgress
		if cluster.Spec.ControlPlaneConfiguration.UpgradeRolloutStrategy.Type == anywherev1.InPlaceStrategyType {
		upgradeStrategyCondition = anywherev1.InPlaceUpgradeInProgress
		}
		conditions.MarkFalse(cluster, anywherev1.ControlPlaneReadyCondition, upgradeStrategyCondition, clusterv1.ConditionSeverityInfo, "Etcd is not ready")
```